core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss

```diff
@@ -27,7 +27,7 @@
         column-gap: $spacing-4;
         row-gap: $spacing-4;
         display: grid;
-        grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+        grid-template-columns: repeat(auto-fill, 250px);
```

Comment:
Why?

Reply:
That `minmax` was making the bookmark panels resize and therefore making the height jump everytime the table is empty while requesting new data. I just test some sizes and `250px` was kind of the ideal size to not break anything and don't make the sizes jump

Reply:
Ok